### ISSUE TITLE
Enqueue Stylesheets Correctly

### ISSUE THREAD

**Issue by buschschwick** [factor1/factor1_blankwp#1]
Should move stylesheets out of `header.php` and into `functions.php`.

This was originally done to avoid cascade issues, but I believe we are beyond that now.

**Comment by buschschwick** [comment]
Noted on current project: must wrap these in a function to avoid conflict with admin panel. This snippet works correctly. 

```php
// Enque Concatenated Stylesheet
	function theme_styles(){
		wp_enqueue_style('theme-styles', get_template_directory_uri() . '/css/all.css', array(), false , 'all');
	}
	add_action( 'wp_enqueue_scripts', 'theme_styles' );
```

**Comment by buschschwick** [comment]
Implemented in 2.0.0
